ministryofjustice/laa-review-criminal-legal-aid
# CRIMRE-73-copy-ref-num

## Description of change
Allows the user to copy the application reference number to the system clipboard

## Link to relevant ticket
https://dsdmoj.atlassian.net/browse/CRIMRE-73

## Notes for reviewer

## Screenshots of changes (if applicable)

### Before changes:
![image](https://user-images.githubusercontent.com/106679087/227498636-2fe21e63-cf53-486d-9102-c04514cdae0f.png)

### After changes:
![image](https://user-images.githubusercontent.com/106679087/227497687-2db05b45-d788-47a4-a141-0c08fd931722.png)

## How to manually test the feature
- User logs into application
- User clicks on applicants name in the table as shown above in the screenshot in either Your list, All open applications or Closed applications pages.
- User is guided to the applicants page
- User clicks on the copy link to copy the applicants reference number
- User can then paste the applicants reference number by using ctrl+v onto another application or window to verify the feature is working
